correlation/src/test/java/com/cerner/beadledom/correlation/CorrelationIdContextTest.java

```diff
@@ -0,0 +1,130 @@
+package com.cerner.beadledom.correlation;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.cerner.beadledom.correlation.CorrelationIdContext.CorrelationIdCloseable;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+class CorrelationIdContextTest {
```

Comment:
Any reason to not use scala like what we have done for other modules?

Reply:
Yeah, static methods and try-with-resources. I also wanted to give junit 5 a try.

Reply:
I like that we can keep a similar BDD-style output with junit5.